### PR TITLE
[Chore] Setting up Ruby on rails

### PR DESCRIPTION
**Pull request by Noelryn**
## PT Link
https://www.pivotaltracker.com/story/show/160916484

## Task
- Scaffholded Ruby on Rails 
- Cleaned out Comments 

## What we Learned
- How to scaffhold Ruby on Rails
- Start pull requests 
- Read reviews and edit the code to match

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.2.1'
+# Use postgresql as the database for Active Record
+gem 'pg', '>= 0.18', '< 2.0'
+# Use Puma as the app server
+gem 'puma', '~> 3.11'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+# Use Uglifier as compressor for JavaScript assets
```
Comment (MiriamSal) [inline Gemfile]: clean out comments

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```
Comment (MiriamSal) [inline Rakefile]: clean out comments

File: bin/setup [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
```
Comment (MiriamSal) [inline bin/setup]: also here clean comments

File: bin/update [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+include FileUtils
+
+# path to your application root.
+APP_ROOT = File.expand_path('..', __dir__)
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+chdir APP_ROOT do
```
Comment (MiriamSal) [inline bin/update:12]: and here..

**Review by MiriamSal — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (Noelryn) [reply Gemfile]: Done

File: Rakefile (hunk 2 above)
Comment (Noelryn) [reply Rakefile]: Done

File: bin/setup (hunk 3 above)
Comment (Noelryn) [reply bin/setup]: Done

File: bin/update (hunk 4 above)
Comment (Noelryn) [reply bin/update:12]: I think the bin doesn't matter it is only really the file we work with, but good to be kept on my toes.

**Review by MiriamSal — APPROVED** [review]

**Comment by faraznaeem** [comment]
The tasks don't match the actual PR

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Go over your PT story and PR description then we are ready for a merge

**Comment by MiriamSal** [comment]
I have added tasks to pt.
And also cleaned up pr, so now we should be ready to merge.

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**